discord/embeds.py

```diff
@@ -111,10 +111,10 @@ def __init__(self, **kwargs):
             colour = kwargs.get('color', EmptyEmbed)
 
         self.colour = colour
-        self.title = kwargs.get('title', EmptyEmbed)
+        self.title = str(kwargs.get('title', EmptyEmbed))
```

Comment:
Casting `str` to `EmptyEmbed` will make the `embed's` title for example look like this:
![image](https://user-images.githubusercontent.com/78174417/112361608-f6a06300-8ca9-11eb-9c84-a041e52c0c4a.png)

Reply:
how would you suggest I resolve this? I could remove the str here, and put an isinstance check for EmptyEmbed later? Or is there a better way you can think of?